pydantic/_internal/_core_utils.py

```diff
@@ -600,6 +600,6 @@ def pretty_print_core_schema(
 
 
 def validate_core_schema(schema: CoreSchema) -> CoreSchema:
-    if 'PYDANTIC_SKIP_VALIDATING_CORE_SCHEMAS' in os.environ:
-        return schema
-    return _validate_core_schema(schema)
+    if os.environ.get('PYDANTIC_VALIDATE_CORE_SCHEMAS') == 1:
```

Comment:
```suggestion
    if os.getenv('PYDANTIC_VALIDATE_CORE_SCHEMAS'):
```

nit: this follows the same logic for `PYDANTIC_PRIVATE_ALLOW_UNHANDLED_SCHEMA_TYPES` and how CPython handles environment variables (it just checks for the present of the environment variable)

Reply:
Sounds good, cc @davidhewitt, you mentioned this check recently as well